markets/perps-market/contracts/storage/GlobalPerpsMarketConfiguration.sol

```diff
@@ -186,18 +196,23 @@ library GlobalPerpsMarketConfiguration {
         return (referralFees, feeCollectorQuote);
     }
 
-    function updateCollateral(
+    function calculateCollateralLiquidateReward(
         Data storage self,
-        uint128 synthMarketId,
+        uint256 notionalValue
+    ) internal view returns (uint256) {
+        return notionalValue.mulDecimal(self.collateralLiquidateRewardRatioD18);
+    }
+
+    function updateSupportedCollaterals(
+        Data storage self,
+        uint128 collateralId,
```

Comment:
Is this where we update the collateral parameters too? Like scalar and lower and upper limits of the discounts?

Reply:
no this is just updating supported collaterals in the system so when integrators call `getSupportedCollaterals` it tells you which ones are active based on if they have a max set.

Reply:
But how do we reconfigure the other parameters in case of update? re-set them?